web/src/lib/domain-logic/getNavBarBusinessTitle.test.ts

```diff
@@ -141,6 +141,38 @@ describe("getNavBarBusinessTitle", () => {
         });
       });
     });
+
+    describe("Nexus DBA", () => {
+      it("shows unamed dba when the user has an unavailable name", () => {
+        const business = generateBusiness({
+          profileData: generateProfileData({
+            businessPersona: "FOREIGN",
+            foreignBusinessType: "NEXUS",
+            businessName: "test business Name",
+            needsNexusDbaName: true,
+            nexusDbaName: "",
+          }),
+        });
+        const navBarBusinessTitle = getNavBarBusinessTitle(business, IsAuthenticated.TRUE);
+        expect(navBarBusinessTitle).toEqual(`${Config.navigationDefaults.navBarUnnamedDbaBusinessText}`);
+      });
+
+      it("shows the dba name when the user has a valid dba name", () => {
```

Comment:
[pebble] the word "valid" is doing some unspecified work here. what does "valid" mean?

If we mean "when the user has entered a DBA name" then we should just say that.